src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* [Instrumentation.Process] Removed CPU utilization metric.
```

Comment:
We are in the Process changelog, so the tag is not needed.

```suggestion
* Removed CPU utilization metric.
```

Reply:
Please put the name of the metric being removed, so it is easier for users looking at the changelog to understand without having to open the PR.

Reply:
fixed; thank you